_dprhtml/features/translations/init.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+class TranslationsViewModel {
+  constructor() {
+    this.downloadTranslation("bt");
+  }
+
+  async downloadTranslation(source) {
+    await DPR_PAL.addOneJS(`/translations/${source}/translations_list.js`);
+    caches.open(`translation-${source}`).then((cache) => this.addTranslToCache(cache, source));
+  }
+
+  async addTranslToCache(cache, source) {
+    let translArray = Array();
+    translArray["bt"] = DPR_G.btUrlsToPrefetch;
+    // add here other transl. sources: ati, abt, dt
```

Comment:
abt is not required as ven. bdhrs just asked for the others.

just bt, dt and ati. so you will need to create similar infrastructure for those.

Reply:
As discussed, for now bt only for testing purposes.